.circleci/scripts/cherry-picker.sh

```diff
@@ -45,11 +51,22 @@ function cherry_pick_with_slack_notification {
             } \
         ] \
         }" "${CONSUL_SLACK_WEBHOOK_URL}"
+
+        # post PR comment to GitHub
+        github_message=":cherries::x: Cherry pick of commit ${commit} onto \`$branch\` failed! [Build Log]($CIRCLE_BUILD_URL)"
```

Comment:
Is there any way we can link to the original PR instead of just the commit, or is that context gone by this point?